js/components/graph/Node.js

```diff
@@ -120,8 +120,13 @@ export default class Node extends React.Component {
         localStorage.setItem(nodeId, newState)
         this.props.childs.forEach(function (node) {
           var currentNode = refLookUp(node, svg)
-          if (currentNode !== undefined) {
```

Comment:
Don't change this code. Instead, define an `updateNode` function in `Bool` that simply calls `this.props.updateNode()`. Similarly for `focusPrereqs` below.